server/data_common/data_adaptor.py

```diff
@@ -21,7 +21,9 @@ def __init__(self, config):
         if config is None:
```

Comment:
Can we clean up the need for test backward compat (ie, fix the tests)?  Completely OK if it is done in a later cleanup PR, just don't want to forget about it (file an issue)

Reply:
I might as well do that.   I've already had to modify all the tests.